File: Test/AdventureWorksFunctionalModel/Production/ScrapReason.cs
namespace AW.Types;

[Bounded]
public class ScrapReason
{

    [Hidden]
    public short ScrapReasonID { get; init; }

    public string Name { get; init; } = "";

    [MemberOrder(99), Versioned]
    public DateTime ModifiedDate { get; init; }

    public override string ToString() => Name;
}

File: Test/AdventureWorksLegacyModelVB/Production/ScrapReason.vb
Namespace AW.Types

	<Bounded>
	Partial Public Class ScrapReason

		<Hidden>
		Public Property ScrapReasonID() As Short

		Public Property Name() As String = ""

		<MemberOrder(99)>
		Public Property ModifiedDate() As DateTime

		Public Overrides Function ToString() As String
			Return Name
		End Function
	End Class
End Namespace